.github/workflows/github.yml

```diff
@@ -23,12 +23,22 @@ jobs:
             os: ubuntu-20.04
             test: 0
             preset: linux-gcc-release
-          - platform: mac
-            os: macos-10.15
+          - platform: mac-intel
+            os: macos-12
```

Comment:
Will resulting builds compatible with catalina?

Reply:
Sure! They're compatible down to 10.13. You can already grab artifact from the github actions run.